text.cabal

```diff
@@ -258,7 +265,7 @@ source-repository head
 test-suite tests
   type:           exitcode-stdio-1.0
   ghc-options:
-    -Wall -threaded -rtsopts
+    -Wall -threaded -rtsopts -with-rtsopts=-N
```

Comment:
Let me know if there's a good reason this wasn't already here. It's not related to the PR.